readme_docker.md

```diff
@@ -0,0 +1,24 @@
+README File For Running Docker Container
```

Comment:
I think that it would be much better to combine this and the Docker section in `README.md` under one subsection in `README.md` since it's not likely that users would be searching for a file under this name. Also, since this information is very helpful, it makes sense to add it to the repository README